README.md

```diff
@@ -1,38 +1,67 @@
-# Website
+# Edit The Docs
+
+To get started, fork and clone the rancher-docs repository.
+
+Our repository doesn't allow you to make changes directly to the `main` branch. Create a working branch and make pull requests from your fork to [rancher/rancher-docs](https://github.com/rancher/rancher-docs). 
+
+## Style & Formatting
+
+The docs are written in [Markdown](https://www.markdownguide.org/cheat-sheet/). We refer to the Microsoft [style guide](https://learn.microsoft.com/en-us/style-guide/welcome/) and generally use standard American English. Many pages are also available in Standard Chinese. We plan to add more language support.
+
+Every docs page contain metadata in the first few lines:
+
+```
+---
+title: Some Title
+---
+```
+
+The `title` is rendered as the page's headline. The renderer wraps the `title` value in H1 tags, which are equivalent to `#` in Markdown syntax. This means that all subsequent headers on the page should be second level (`##`) or more.
+
+## Organization
+
+Directories in the repo correspond to submenus in the site sidebar. We try to keep our submenus to a maximum of three levels deep, or four if absolutely necessary.
+
+The sidebar on the live site is rendered based on the contents of the file, `sidebars.json`, which is located in the top level of the repository. If you add, move, or delete a page, `sidebars.json` must be updated.
```

Comment:
If the idea is to keep the info brief, `sidebars.js` is the file that should be called out since that is used to manage the latest/current version of the site's content, which is where most contributions are made.

Otherwise, additional clarification is needed here as the sidebar file used depends on the context.
- `sidebars.js` is for content under `/docs`. Files under `/docs` corresponds to our latest/current content.
- `version-XYZ-sidebars.json` is associated with content under `/versioned_docs`, which is for older versions of our content.

Reply:
Maybe we should remove this section -- on a practical level, internal contributors are probably going to be the ones moving, adding, or deleting pages.

Reply:
Sure, let's remove this section.